test/docker/docker-compose.yml

```diff
@@ -87,12 +88,27 @@ services:
     container_name: tigris_server2
     image: tigris_server
     environment:
+      - TIGRIS_SERVER_SERVER_PORT=8082
       - TIGRIS_ENVIRONMENT=test
       - TIGRIS_SERVER_SEARCH_AUTH_KEY=ts_test_key
       - TIGRIS_SERVER_SEARCH_HOST=tigris_search
       - TIGRIS_SERVER_LOG_FORMAT=console
       - TIGRIS_SERVER_CDC_ENABLED=true
       - TIGRIS_SERVER_SECONDARY_INDEX_WRITE_ENABLED=true
+      - TIGRIS_SERVER_AUTH_ENABLED=true
+      - TIGRIS_SERVER_AUTH_ENABLE_NAMESPACE_ISOLATION=true
```

Comment:
What does this do?

Reply:
Enable namespace isolation makes tigris set the namespace in context of an operation. Enable auth enables the authentication requirement for Tigris.

Reply:
Don't we need both to be enabled in a dependent way?

Reply:
You can have auth and still all request falls into default (shared) namespace.